buildenv/Build_Instructions_V11.md

```diff
@@ -0,0 +1,457 @@
+
+:pencil: If you require a heap size greater than 57GB, enable a noncompressedrefs build with the `--with-noncompressedrefs` option during this step.
+
+### 4. build
+:blue_book:
+Now you're ready to build OpenJDK with OpenJ9:
+```
+make all
+```
+:warning: If you just type `make`, rather than `make all` your build will fail, because the default `make` target is `exploded-image`. If you want to specify `make` instead of `make all`, you must add `--default-make-target=images` when you run the configure script. For more information, read this [issue](https://github.com/ibmruntimes/openj9-openjdk-jdk9/issues/34).
+
+Two Java builds are produced: a full developer kit (jdk) and a runtime environment (jre)
+- **build/aix-ppc64-normal-server-release/images/jdk**
+- **build/aix-ppc64-normal-server-release/images/jre**
+
+### 5. Test
+:blue_book:
+For a simple test, try running the `java -version` command.
+Change to the /jdk directory:
+```
+cd build/aix-ppc64-normal-server-release/images/jdk
+```
+Run:
+```
+./bin/java -version
+```
+
+Here is some sample output:
+
+```
+openjdk version "10-internal" 2018-03-20
+OpenJDK Runtime Environment (build 10-internal+0-adhoc..openj9-openjdk-jdk10)
+Eclipse OpenJ9 VM (build master-4ecf784, JRE 10 AIX ppc64-64 Compressed References 20180411_000000 (JIT enabled, AOT enabled)
+OpenJ9   - 4ecf784
+OMR      - cb9e24d
+JCL      - eaa06eb based on jdk-10+46)
+```
+:blue_book: *Congratulations!* :tada:
+
+----------------------------------
+
+## Windows
+:ledger:
+
+The following instructions guide you through the process of building a Windows **OpenJDK V11** binary that contains Eclipse OpenJ9. This process can be used to build binaries for Windows 7, 8, and 10.
+
+### 1. Prepare your system
+:ledger:
+You must install a number of software dependencies to create a suitable build environment on your system:
+
+- [Cygwin](https://cygwin.com/install.html), which provides a Unix-style command line interface. Install all packages in the `Devel` category. Included in this package is mingw, a minimalist subset of GNU tools for Microsoft Windows. OpenJ9 uses the mingw/GCC compiler and is tested with version 6.4.0 of same. Other versions are expected to work also. In the `Archive` category, install the packages `zip` and `unzip`. In the `Utils` category, install the `cpio` package. Install any further package dependencies that are identified by the installer. More information about using Cygwin can be found [here](https://cygwin.com/docs.html).
+- [Windows JDK 10](https://adoptopenjdk.net/releases.html?variant=openjdk10#x64_win), which is used as the boot JDK.
+- [Microsoft Visual Studio 2013]( https://go.microsoft.com/fwlink/?LinkId=532495), which is the same compiler level used by OpenJDK; or [Microsoft Visual Studio 2017]( https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community&rel=15), which is now supported.
```

Comment:
Does 2013 work with Java 11?

Reply:
It seems 2017 is the default for Java 11? If so "which is the same compiler level used by OpenJDK" isn't appropriate.

Reply:
According to openj9-openjdk-jdk11_v1/make/autoconf/toolchain_windows.m4:
```
# The order of these defines the priority by which we try to find them.
VALID_VS_VERSIONS="2017 2013 2015 2012 2010"
...
```
If --with-toolchain-version is not supplied on the command line, it searches the VS version from 2017 by order in VALID_VS_VERSIONS by default,  depending on which VS version is installed on Windows. So it looks for VS2013 if VS2017 doesn't exist.

>It seems 2017 is the default for Java 11
Pretty much yes in such case. I already updated the description there.